src/cai_lab_to_nwb/zaki_2024/zaki_2024_convert_week_session.py

```diff
@@ -49,6 +46,20 @@ def session_to_nwb(
     )
     conversion_options.update(dict(MultiEDFSignals=dict(stub_test=stub_test)))
 
+    # Add Cross session cell registration
+    main_folder = data_dir_path / f"Ca_EEG_Calcium/{subject_id}/SpatialFootprints"
+    file_paths = []
+
+    for folder in main_folder.iterdir():
```

Comment:
What do you think about throwing errors when the expected folder_paths and the file_paths are not found? right now they are skipped silently but if the point of this interface is that those things should be added maybe it is better to stop the process at the tracks and throw an informative error so the user is aware of this.

I am not sure, maybe you have a good reason to skip instead of stopping.

Reply:
right! I added few lines to throw an error only when there is no file in the specific folder (matching the pattern) as expected. The folder.is_dir() "check" is just to select the folder paths among all the type Paths objects iterdir might iterate over (both folders and files), thus no need to throw an error if that object is not a directory.